packages/vc-verification/src/claim-issuer-verification.ts

```diff
@@ -127,11 +127,11 @@ export class ClaimIssuerVerification {
       namespace
     );
     if (issuers && issuers.did && issuers.issuerType === 'DID') {
-      for (let i = 0; i < issuers.did.length; i++) {
-        if (issuers.did[i] === issuerDID) {
-          return true;
-        }
-      }
+      return issuers?.did?.find(
+        (d) => addressOf(d).toUpperCase() === addressOf(issuerDID).toUpperCase()
+      )
+        ? true
+        : false;
```

Comment:
Does this also need to be changed in `vc-issuer-verification`?

I think similar to my other question, this might be creating a strange API where sometime it returns `false` on failed verification and sometimes it throws an error